CLAUDE.md

```
@@ -0,0 +1,78 @@
+go test -v -run TestContainersCommandConcurrency ./docker/
+
+# Install/update dependencies
+go mod download
+go mod tidy
+```
+
+## Architecture
+
+The codebase follows a layered architecture with channel-based communication between layers:
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)